Objects/Objects/BuiltElements/Revit/RevitCeiling.cs

```diff
@@ -62,7 +66,7 @@ public class RevitCeiling : Ceiling
   public double slope { get; set; }
   public Line? slopeDirection { get; set; }
 
-  [Obsolete("Offset property is now captured in parameters to match the behavior of other Revit objects")]
+  [Obsolete("Offset property is now captured in parameters to match the behavior of other Revit objects", true)]
```

Comment:
Are we happy with this being `error: true`?

Reply:
I think yes in this case, given that there were no references to `offset` in the converters